WISE-Community/WISE-Client
# chore(NodeAuthoring): UI improvements

## Changes
- Move import component option to the "add component choose component types" view
- Remove Preview without constraints button
- Remove EditAdvancedComponentAngularJSController.ts - no longer used

## Test (in Node Authoring view)
- Select new component view
   - Now shows option to import component
      - Importing component feature works, including Back/Cancel buttons
   - Select new component works as before
- Preview node without constraint button is no longer displayed. Preview node button works as before. 

Closes #1317
